packages/uploadthing/src/_internal/upload-browser.ts

```diff
@@ -63,7 +63,9 @@ const uploadWithProgress = (
 
     // Is there a case when the client would throw and
     // ingest server not knowing about it? idts?
-    xhr.addEventListener("error", () => {
+    xhr.addEventListener("error", (err) => {
+      // eslint-disable-next-line no-console
+      console.log("XHR error", err, xhr);
       resume(
         new UploadThingError({
           code: "UPLOAD_FAILED",
```

Comment:
Should this error also include the cause?

Reply:
this PR can be closed since there's no way for JS to actaully get the error here: https://issues.chromium.org/issues/40170162#comment6